### PR TITLE
[vLLM] QBits Perf Enhence

### PR DESCRIPTION
**Pull request by Zhenzhong1**
## Type of Change

* Added customized config & updated benchmark script 
* Fixed the vLLM-qbits 1024 in,32 out next token issues

## Description

vLLM perf:
image

image

## Expected Behavior & Potential Risk

N/A

## How has this PR been tested?

Manually profiling.

## Dependency Change?

N/A

### CODE REVIEW

**Review by changwangss — APPROVED** [review]

File: intel_extension_for_transformers/transformers/llm/quantization/nn/modules.py [hunk 1]
```diff
@@ -146,8 +146,10 @@ def forward(self, x: torch.Tensor):
         bias = None if self.bias is None else self.bias.data.float()
         if not x.is_contiguous():
             x = x.contiguous()
+
+        # QBits only supports the FP32 activation dtype currently.
```
Comment (zhewang1-intc) [inline intel_extension_for_transformers/transformers/llm/quantization/nn/modules.py:150]: only fp32 activation support gemv which benefits next-token.
leave a todo here? QBits(BesTLA) will support gemv with bf16 activation very soon

**Review by zhewang1-intc — APPROVED** [review]

File: intel_extension_for_transformers/transformers/llm/quantization/nn/modules.py (hunk 1 above)
Comment (Zhenzhong1) [reply intel_extension_for_transformers/transformers/llm/quantization/nn/modules.py:150]: fixed

**Comment by Zhenzhong1** [comment]
@XuehaoSun Hi, if this `Optimize Unit Test ` CI issue fixed, please let me know and re-run the CI. Thx.

**Comment by Zhenzhong1** [comment]
ready for merge.

**Pull request merged**